components/eam/tools/topo_tool/cube_to_target/remap.F90

```diff
@@ -850,7 +850,8 @@ subroutine side_integral(&
 
             if (xseg(1).EQ.xseg(2))then
               slope = bignum
-            else if (abs(yseg(1) -yseg(2))<fuzzy_width) then
+            !else if (abs(yseg(1) -yseg(2))<fuzzy_width) then !remove the exact integration
```

Comment:
@jsbamboo let's change the wording on the comment on both of the commented lines to something more verbose and clear, I'm thinking: 
"_this will enable exact integration, which was found to create problematic negative weights_"

A better solution would be to just add a logical switch controlled by a flag, but I'm fine with this solution as well.